modell/src/main/kotlin/no/nav/tilbakekreving/hendelse/KravgrunnlagHendelse.kt

```diff
@@ -18,16 +19,20 @@ class KravgrunnlagHendelse(
     private val ansvarligEnhet: String,
     private val kontrollfelt: String,
     // Brukes som eksternId i henting av fagsysteminfo, hva betyr det egentlig?
-    private val referanse: String,
+    val referanse: String,
     private val kravgrunnlagId: String,
-    private val perioder: List<Periode>,
+    val perioder: List<Periode>,
 ) : Historikk.HistorikkInnslag<UUID> {
     fun totaltBeløpFor(periode: Datoperiode): BigDecimal =
         perioder.single { kgPeriode -> kgPeriode.inneholder(periode) }
             .totaltBeløp()
 
     fun datoperioder() = perioder.map { it.periode }
 
+    fun totalFeilutbetalBeløpForAllePerioder() = perioder.sumOf { it.totaltBeløp() }
```

Comment:
er dette nok kanskje?
```suggestion
    fun feilutbetaltBeløpForAllePerioder() = perioder.sumOf { it.totaltBeløp() }
```

Reply:
metoden brukes ikke lenger så det er fjernet.